src/components/note/__internal__/status-icon/status.spec.tsx

```diff
@@ -1,16 +1,16 @@
 import React from "react";
```

Comment:
Same as above -> rename to `status-icon.spec.tsx`

Reply:
changed